src/sentry/api/bases/organization.py

```diff
@@ -137,20 +137,10 @@ def get_project_ids(
 
         requested_projects = project_ids.copy()
 
-        om_role = None
         user = getattr(request, 'user', None)
-        if user and user.is_authenticated():
-            try:
-                om_role = OrganizationMember.objects.filter(
-                    user=request.user,
-                    organization=organization,
-                ).values_list('role', flat=True).get()
-            except OrganizationMember.DoesNotExist:
-                pass
 
         if (
-            user and user.is_superuser
-            or (om_role and roles.get(om_role).is_global)
+            user and is_active_superuser(request)
```

Comment:
Dang, we use `user.is_superuser` a fair bit around the codebase... wonder if we need to check the usage at some point.

Reply:
yeah i will look through them and fix the necessary ones in a follow up pr